### ISSUE TITLE
[FEATURE REQUEST] Add an icon for the website as favicon.ico

### ISSUE THREAD

**Issue by AVS1508** [umass-cicsoft/umass-cicsoft.github.io#27]
### **Is your feature request related to a problem? Please describe.**
Currently, the website title has no accompanying icon, thus, we should add an icon with the CICSoft logo.

### **Describe the solution you would like**
Place the `favicon.ico` from here ([favicon.zip](https://github.com/umass-cicsoft/umass-cicsoft.github.io/files/7794396/favicon.zip)) into the root `/` folder, and add the following inside the `head` tag in `index.html`:

```HTML
<head>
    ...
    <link rel="icon" href="favicon.ico" />
    ...
</head>
```

### **Describe alternatives you have considered**
Placement of `favicon.ico` could be into the `assets/icons` folder but that is not recommended.

### **Additional Context**
Reach out to @AVS1508 if you have any queries.

**Comment by m-abdollahi** [comment]
already done! @AVS1508

**Comment by AVS1508** [comment]
This issue is resolved through merging #30.
